tests/components/test_logbook.py

```diff
@@ -9,25 +9,63 @@
 from datetime import timedelta
 
 import homeassistant.core as ha
+from homeassistant.components.logbook import Entry
 from homeassistant.const import (
     EVENT_STATE_CHANGED, EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.util.dt as dt_util
 from homeassistant.components import logbook
 
-from tests.common import get_test_home_assistant, mock_http_component
+from tests.common import mock_http_component
 
 
 class TestComponentHistory(unittest.TestCase):
     """ Tests homeassistant.components.history module. """
 
-    def test_setup(self):
+    def setUp(self):
         """ Test setup method. """
-        try:
-            hass = get_test_home_assistant()
-            mock_http_component(hass)
-            self.assertTrue(logbook.setup(hass, {}))
-        finally:
-            hass.stop()
+        self.hass = ha.HomeAssistant()
+        mock_http_component(self.hass)
+        logbook.setup(self.hass, {})
+
+        self.calls = []
+
+    def tearDown(self):
+        self.hass.stop()
+
+    def test_service_call_create_logbook_entry(self):
+        calls = []
+
+        def event_listener(event):
+            calls.append(event)
+
+        self.hass.bus.listen(logbook.EVENT_LOGBOOK_ENTRY, event_listener)
+        self.hass.services.call('logbook', 'log', {
+            logbook.ATTR_NAME: 'name',
+            logbook.ATTR_MESSAGE: 'message',
+            logbook.ATTR_DOMAIN_INPUT: 'domain',
+            logbook.ATTR_ENTITY_ID: 'entity_id',
+        }, True)
+        self.hass.pool.block_till_done()
+
+        self.assertEqual(1, len(calls))
+        last_call = calls[-1]
+
+        self.assertEqual(last_call.data.get('name'), 'name')
```

Comment:
The order of arguments for assertEqual is `assertEqual(expected, value_to_test)` so please change the order of your arguments.